packages/stream_video/lib/src/stream_video.dart

```diff
@@ -572,13 +572,18 @@ class StreamVideo {
     final callCid = payload['call_cid'] as String?;
     if (callCid == null) return false;
 
-    final uuid = const Uuid().v4();
+    var callId = const Uuid().v4();
+    final splitCid = callCid.split(':');
+    if (splitCid.length == 2) {
+      callId = splitCid.last;
+    }
+
```

Comment:
Are we sure it's in the correct format? I remember this giving me some troubles earlier.

Reply:
yes, the issue is with iOS CallKit requiring it to be valid uuid, which it is when we split it and use the id part and not the whole cid

Reply:
For some reason, in the early video time it sometimes wasn't a valid UUID, good to know it is now. This is great, thanks.